utils/analytics.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  getAnalytics,
+  getAppInstanceId,
+  logEvent,
+  setAnalyticsCollectionEnabled,
+  logScreenView,
+} from '@react-native-firebase/analytics';
+import { getApp } from '@react-native-firebase/app';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+let analytics: ReturnType<typeof getAnalytics> | null = null;
+try {
+  const app = getApp();
+  analytics = getAnalytics(app);
+} catch (error) {
+  // Analytics initialization failed - app will continue without analytics
+}
+const sanitize = (value?: string): string =>
+  typeof value === 'string' && value.trim().length > 0 ? value.trim() : 'undefined';
```

Comment:
It returns the literal string 'undefined' for blank input. Prefer skipping the parameter or using a clearer fallback like 'unknown'. Logging 'undefined' can degrade analytics quality.